docs/usage/config-presets.md

```diff
@@ -232,6 +232,25 @@ Parameters are supported similar to other methods:
 }
 ```
 
+## Templating presets
+
+In order to provide flexible preset configurations, Renovate provides support using [templates](./templates.md) for preset names.
+
+### Example use-case
+
+The following example shows a self-hosted Renovate configuration for a company called "Renovators".
+The company is split into multiple organisations `foo`, `bar` and `baz`, where each organisation provides its opinionated shareable preset.
+Renovate is configured to load the preset of each organisation dynamically based on the `ORGANISATION_SLUG`environment variable.
```

Comment:
```suggestion
Renovate is configured to load the preset of each organisation dynamically based on the `ORGANIZATION_SLUG`environment variable.
```

We use American English, which uses a z for words like organization. Please use the z form in the docs and the code.